### DISCUSSION TITLE
[v2.3.0-beta.1] Question re Global CSS

### DISCUSSION THREAD

**Discussion by willliamchan** [Clooos/Bubble-Card#795] [Questions about config, custom styles and templates]
Hi Clooos

Before we move forward, can I check if there is any plan to make Global CSS support per domain (entity type)?

I am dreaming that I can define CSS per type at some point like one set of CSS for lock, another set of CSS for blind... etc.

Cheers
W

**Comment by Clooos** [comment]
Hi, I will probably not add that possibility, just because it would need some more JS code that could potentially slow down the code for a too small use case.

But you can create these variables yourself in your custom styles. For example:

```
.bubble-icon-container {
  background: var(--bubble-lock-icon-color) !important;
}
```

Then define this variable in your theme file.

This need some manual work but it allows you to change them easily after that.

**Reply by landocarlesien** [reply]
Can you provide an example of how this would look in the theme?

**Comment by willliamchan** [comment]
By any chance you can make a full example? Maybe another YouTube video for that later.

No rush at all of coz I am fully set for my dashboard, just thinking of ways to optimise them when thing change.

________________________________

**Reply by Clooos** [reply]
My next video will be about custom styles, I will try to add this example in it 🙂

**Comment by willliamchan** [comment]
That would be very helpful!
________________________________
